dart/config.hpp.in

```diff
@@ -28,6 +28,18 @@
   (DART_MAJOR_VERSION < x || (DART_MAJOR_VERSION <= x && \
   (DART_MINOR_VERSION < y || (DART_MINOR_VERSION <= y))))
 
+/* urdfdom_headers Version number */
+// We define the version numbers of urdfdom_headers here since it doesn't expose
+// the version numbers itself in source level.
+#define URDFDOM_HEADERS_MAJOR_VERSION @urdfdom_headers_VERSION_MAJOR@
+#define URDFDOM_HEADERS_MINOR_VERSION @urdfdom_headers_VERSION_MINOR@
+#define URDFDOM_HEADERS_PATCH_VERSION @urdfdom_headers_VERSION_PATCH@
+
+#define URDFDOM_HEADERS_VERSION_AT_LEAST(x,y,z) \
+  (URDFDOM_HEADERS_MAJOR_VERSION > x || (URDFDOM_HEADERS_MAJOR_VERSION >= x && \
+  (URDFDOM_HEADERS_MINOR_VERSION > y || (URDFDOM_HEADERS_MINOR_VERSION >= y && \
+  URDFDOM_HEADERS_PATCH_VERSION >= z))))
```

Comment:
It actually has `URDFDOM_HEADERS_MAJOR_VERSION >= x`. If it is reached to the last part (i.e., `URDFDOM_HEADERS_PATCH_VERSION >= z`), that means it's the case of `URDFDOM_HEADERS_MAJOR_VERSION >= x` AND `URDFDOM_HEADERS_MINOR_VERSION >= y`.

The logic expressed in C++ would be like:
```cpp
if (major > x)
{
  return true;
}
else if (major >= x)
{
  if (minor > y)
  {
    return true;
  }
  else if (minor >= y)
  {
    // the third part
    if (patch >= z)
      return true;
    else
      return false;
  }
  else
  {
    return false;
  }
}
else
{
  return false;
}
```

I borrowed this from `Eigen`, but yeah it's pretty confusing.